pkg/importer/data-processor.go

```diff
@@ -264,16 +264,32 @@ func (dp *DataProcessor) convert(url *url.URL) (ProcessingPhase, error) {
 }
 
 func (dp *DataProcessor) resize() (ProcessingPhase, error) {
-	// Resize only if we have a resize request, and if the image is on a file system pvc.
 	size, _ := getAvailableSpaceBlockFunc(dp.dataFile)
 	klog.V(3).Infof("Available space in dataFile: %d", size)
+	isBlockDev := size >= int64(0)
```

Comment:
// size will be int64(-1) for non-block dev

Reply:
yeah, I left that comment out in favour of using a descriptive name for this condition :-)